e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,184 @@
+import UserAdminPage from 'app/page/admin-user-list-page';
+import { signInWithAccessToken } from 'utils/test-utils';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+import UserProfileInfo from 'app/page/admin-user-profile-info';
+
+describe('Admin', () => {
+  const userEmail = 'admin_test';
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page);
+    await navigation.navMenu(page, NavLink.USER_ADMIN);
+  });
+
+  test('verify admin-user-profile page reflect the same modules and active status as on admin-table', async () => {
```

Comment:
@aweng98 I've split the previous test case into 2: 

1. this one to verify the bypass status on user-admin-table vs user-profile page.
2. second one : to verify all the other fields on user-admin-table vs user-profile page.